api/api.go

```diff
@@ -513,29 +513,43 @@ func WhoamiQuery(cl *graphql.Client) (*WhoamiResponse, error) {
 }
 
 // ConfigQuery calls the GQL API to validate and process config
-func ConfigQuery(cl *graphql.Client, configPath string, pipelineValues pipeline.Values) (*ConfigResponse, error) {
+func ConfigQuery(cl *graphql.Client, configPath string, orgSlug string, pipelineValues pipeline.Values) (*ConfigResponse, error) {
 	var response BuildConfigResponse
+	var query string
 
 	config, err := loadYaml(configPath)
 	if err != nil {
 		return nil, err
 	}
 
-	query := `
-		query ValidateConfig ($config: String!, $pipelineValues: [StringKeyVal!]) {
-			buildConfig(configYaml: $config, pipelineValues: $pipelineValues) {
-				valid,
-				errors { message },
-				sourceYaml,
-				outputYaml
-			}
-		}`
+	if orgSlug != "" {
+		query = `query ValidateConfig ($config: String!, $pipelineValues: [StringKeyVal!], $orgSlug: String!) {
```

Comment:
Is `orgSlug` an optional `String`, according to the schema?

Reply:
Good catch, it is!